src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts

```diff
@@ -100,8 +100,9 @@ export const EDIT_PAGE_CSS = `
 
     .dotedit-container__toolbar {
         position: relative;
-        margin: -20px 8px 0 0;
+        margin: -15px 8px 0 0;
         float: right;
+        font-size: 0;
```

Comment:
What is this for?

Reply:
display the `+` button correctly in the containers, for empty templates or any other.